### DISCUSSION TITLE
Clickhouse Kafka engine

### DISCUSSION THREAD

**Discussion by jcolabre0514** [ClickHouse/ClickHouse#64629] [Q&A]
I am trying to setup a connection to my kafka from Clickhouse. I added this section in my config.xml

<kafka>
<ssl_ca_location>/path/to/cert/cert.pem </ssl_ca_location>
<security_protocol>sasl_ssl</security_protocol>
<sasl_mechanism>PLAIN</sasl_mechanism>
<sasl_username>abc</sasl_mechanism>
<sasl_password>abc</sasl_password>
</kafka>

I am getting sasl authentication failed

May I know if I am missing something in the config?

**Comment by yiyang-shao** [comment]
Hi! 

Could you provide an example of your `config.xml` file on how you added the config?

**Reply by jcolabre0514** [reply]
![image](https://github.com/ClickHouse/ClickHouse/assets/167754954/435bc19e-f37f-4d44-a75f-a4b9da28c5bc)

**Reply by jcolabre0514** [reply]
The screenshot on top is a sample kafka config I made in config.xml. Just a typo on the / start of the clickhouse section.
